src/cli/flags/options.go

```diff
@@ -9,6 +9,7 @@ const (
 	DisableConcurrencyLimiterFN = "disable-concurrency-limiter"
 	DisableDeltaFN              = "disable-delta"
 	DisableIncrementalsFN       = "disable-incrementals"
+	DisableAssistCachingFN      = "disable-caching"
```

Comment:
I worry about this is overly generic, which makes it difficult to grok and potentially collides with other (maybe future) forms of caching.  Not sure there's some foo for `disable-foo` that won't run into similar problems in this case.  Perhaps some version of `force-foo` would communicate the intent more clearly.  `force-full` or `force-clean` or `force-new`.